playbooks/docker-machine-functional-public-clouds/run.yaml

```diff
@@ -0,0 +1,47 @@
+- hosts: all
+  become: yes
+  roles:
+    - export-cloud-openrc
+    - config-docker-machine
+  tasks:
+    - name: Create Docker machine
+      shell:
+        cmd: |
+          set -ex
+          set -o pipefail
+          git clone https://github.com/sstephenson/bats.git
+          pushd bats
+          ./install.sh /usr/local
+          popd
+          # install Docker
+          apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D || true
+          apt-key adv --keyserver hkp://pgp.mit.edu:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D || true
+          apt-add-repository 'deb http://apt.dockerproject.org/repo ubuntu-xenial main'
+          apt-get update
+          apt-cache policy docker-engine
+          apt-get install -y docker-engine=1.12.6-0~ubuntu-xenial
+          cat /lib/systemd/system/docker.service
+          sed -r -i "s|(ExecStart)=(.+)|\1=\2 --iptables=false|" /lib/systemd/system/docker.service
+          cat /lib/systemd/system/docker.service
+          systemctl daemon-reload
+          systemctl restart docker
+          ifconfig -a
```

Comment:
We don't have to install docker-engine into docker-machine executor vm, because we use OpenStack driver and test against public cloud resources, all of docker environment will be applied into public cloud vms

Reply:
let me test again, I remember it is necessary

Reply:
after testing, in running integration tests, it will raise `docker: command not found` if we don't install docker

Reply:
Yeah, I saw testing log, some integration test cases depend on `docker` command, thank you.